lib/chef/mixin/which.rb

```diff
@@ -19,6 +19,10 @@ class Chef
   module Mixin
     module Which
       def which(cmd, opts = {})
+        Chef::Mixin::Which.which(cmd, opts)
+      end
+
+      def self.which(cmd, opts = {})
```

Comment:
i think i'm saying you can leave the module alone and use extend instead of include to mix the module into a class and it'll solve this problem.

i just can't quite figure out how this is even working for you right now, though, because i think the class method in the module will get ignored...    which is to say that having `def self.foo` in a module usually doesn't work at all...